drivers/led/sn32f2xx.c

```diff
@@ -380,7 +500,7 @@ static void update_pwm_channels(PWMDriver *pwmp) {
     for (uint8_t current_key_row = 0; current_key_row < MATRIX_ROWS; current_key_row++) {
         uint8_t led_index = g_led_config.matrix_co[current_key_row][current_key_col];
 #    if (SN32_PWM_CONTROL == SOFTWARE_PWM)
-        if (led_index >= SN32F24XB_LED_COUNT) continue;
+        if (led_index >= SN32F2XX_LED_COUNT) continue;
```

Comment:
For me this line is necessary also for hardware PWM.
If this is not used the last keys in cols which have less then the standard amount of keys.  (e.g. just 4 than normally 5) will light up dimly.